examples/kubeval/simple/README.md

```diff
@@ -1,35 +1,65 @@
 # kubeval: Simple Example
 
-The `kubeval` KRM config function validates Kubernetes resources using kubeval.
-Learn more on the [kubeval website].
+### Overview
 
-This example invokes the kubeval function against the builtin Kubernetes
-v1.19.8 schema.
+This example demonstrates how to declaratively run [`kubeval`] function to
+validate KRM resources.
+
+The following is the `Kptfile` in this example: 
+
+```yaml
+apiVersion: kpt.dev/v1alpha2
+kind: Kptfile
+metadata:
+  name: example
+pipeline:
+  validators:
+    - image: gcr.io/kpt-fn/kubeval:unstable
+      configMap:
+        strict: 'true'
+```
+
+The function configuration is provided using a ConfigMap. We set 1 key-value
+pair:
+- `strict: 'true'`: We disallow unknown fields.
 
 ### Function invocation
 
 Get this example and try it out by running the following commands:
 
-<!-- @getAndRunPkg @test -->
 ```sh
 kpt pkg get https://github.com/GoogleContainerTools/kpt-functions-catalog.git/examples/kubeval/simple .
-kpt fn run simple
+kpt fn render simple
 ```
 
 ### Expected Results
 
 This should give the following output:
 
 ```sh
-[ERROR] Additional property templates is not allowed in object 'v1/ReplicationController//bob' in file resources.yaml in field templates
-[ERROR] Invalid type. Expected: [integer,null], given: string in object 'v1/ReplicationController//bob' in file resources.yaml in field spec.replicas
-error: exit status 1
+Package "simple": 
```

Comment:
I would recommend to showing the the non-structured output which is likely to change (no backwards compatibility).